docs/assets/css/custom.css

```
@@ -0,0 +1,23 @@
+.logo-config {
+  vertical-align: middle !important;
+  width: 100%;
+  height: auto;
+  width: calc(100px + 6 * ((100vw - 320px) / 680))
+}
+
+.c-footer-style {
+  padding: 20px;
+  background: #333333;
+  color: #ffffff;
+}
+.footer-heading-row {
+  font-size: 21px !important;
+  font-weight: bold;
+  margin-bottom: calc(40px + (48 - 40) * ((100vw - 360px) / (1600 - 360))) !important;
+}
+
+.navbar-brand:hover{
+  text-decoration: underline;
+}
```

I'll leave it up to you exactly how you want to handle the underlining of nav links, but it would be nice if we are consistent. This seems to be adding underline on hover for the central link, but the others are always underlined. (Bootstrap seems to be underlining a tags by default in regular text or the footer?). I'd probably just keep it consistent and do something like this, removing the `<u>` tags from `nav.html`:

```suggestion
.navbar-brand:hover, .navbar-link:hover {
  text-decoration: underline;
}
```